backend/src/main/java/com/example/backend/global/jwt/filter/JwtFilter.java

```diff
@@ -0,0 +1,71 @@
+package com.example.backend.global.jwt.filter;
+
+import com.example.backend.global.jwt.JwtProvider;
+import com.example.backend.global.jwt.exception.LoggedOutTokenException;
+import com.example.backend.global.jwt.refreshtoken.repository.RefreshTokenRedisRepository;
+import com.example.backend.global.jwt.response.JwtValidateResponse;
+import com.example.backend.global.jwt.service.RedisService;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+@Component
+@RequiredArgsConstructor
+public class JwtFilter extends OncePerRequestFilter {
+
+    private static final String AUTHORIZATION_HEADER = "Authorization";
+    private final JwtProvider jwtProvider;
+    private final RedisService redisService;
+    private final RefreshTokenRedisRepository refreshTokenRedisRepository;
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
+        throws ServletException, IOException {
+
+        if (request.getRequestURI().startsWith("/api/auth/kakao-login")) {
+            filterChain.doFilter(request, response);
+            return;
```

Comment:
필터내부에 있는 shouldNotFilter 메서드 override 해서 처리해주면 좋아보이는데 고민해보면 좋을듯 (정답은 없으니까)

```suggestion
        @Override
        protected boolean shouldNotFilter(HttpServletRequest request) {
            return request.getRequestURI().startsWith("/api/auth/kakao-login");
        }
```